### ISSUE TITLE
Tversky loss implementation

### ISSUE THREAD

**Issue by thegooby** [MIC-DKFZ/nnUNet#2032]
Hello,

I'm trying to improve contouring of the esophagus on lung/chest cases, and playing around with the included losses and focal loss still leave something to be desired. The organ is generally undercontoured instead of overcontoured, so I would like to use the formulation to more negatively penalize false negatives than false positives. 

I have tried playing around with some of the preexisting functions, such as the ones here https://github.com/Mr-TalhaIlyas/Loss-Functions-Package-Tensorflow-Keras-PyTorch , but it isn't quite working as it should. I'm not familiar enough with Python to do a great job of converting the Pytorch version of Tversky loss into one that works with the other nnUNet functions.

If there is an even simpler implementation of this loss that involves another class into the dice.py file, that would work, too. Any help would be greatly appreciated!

**Comment by thegooby** [comment]
In retrospect, this was a silly question. To anyone with a similar question, I just modified the compound_losses file to have a Tversky_and_CE loss, which itself was a copy pasted dice_and_CE loss with slight modifications for consistency. In the trainer file, I changed dice_and_CE to Tversky_and_CE. In the dice.py, I made a Tversky class that is a copy of the softdiceloss with factors in the denominator for it to be Tversky loss.
